LEAF_Request_Portal/js/dialogController.js

```diff
@@ -94,24 +96,32 @@ dialogController.prototype.showButtons = function() {
 };
 
 dialogController.prototype.enableLiveValidation = function() {
-	var t = this;
-	$('input[type="text"]').on('keyup', function() {
+	let t = this;
```

Comment:
Sorry Pete. Missed this the first time.  I know this wasn't you.  Please get rid of all of them (there are at least 3) and fix all the times it was used.  I'm surprised this hasn't bitten up in the behind yet.